driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/recvmmsg.bpf.c

```diff
@@ -34,22 +40,125 @@ int BPF_PROG(recvmmsg_e, struct pt_regs *regs, long id) {
 
 /*=============================== EXIT EVENT ===========================*/
 
-SEC("tp_btf/sys_exit")
-int BPF_PROG(recvmmsg_x, struct pt_regs *regs, long ret) {
-	struct ringbuf_struct ringbuf;
-	if(!ringbuf__reserve_space(&ringbuf, ctx, RECVMMSG_X_SIZE, PPME_SOCKET_RECVMMSG_X)) {
+typedef struct recvmmsg_data_s {
```

Comment:
```suggestion
typedef struct {
```
Less confusing IMHO :)